awsx/scripts/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
```

Comment:
We might want to wait with this one until we have reliable x-platform verification of the binaries.